### DISCUSSION TITLE
Default text size is a little bit smaller for me

### DISCUSSION THREAD

**Discussion by ryota-murakami** [RecordReplay/devtools#6667] [Ideas]
This is come from [Discord general thread](https://discord.com/channels/779097926135054346/779097926135054350/972703384798892082), so I copy & paste original my comment at the time.

#### Original Comment

Hey guys, I'm tracking React internal code path with general behavior using breakpoint & step-in.
Debugging is going as well, but default text size is a little bit smaller for me. 😅
Is there any font size config option?

<img width="1355" alt="Screen Shot 2022-05-08 at 11 41 24" src="https://user-images.githubusercontent.com/5501268/167281978-378884e6-8f49-46c6-aaf0-b83e65f067f5.png">

**Comment by jasonLaster** [comment]
@ryota-murakami would you enjoy taking a stab at this?

I think the desired behavior would be a dropdown in the Appearances panel for font size with (normal, small, large) we could of course do this in px too, but might be easier to start with one or two options...

https://cs.github.com/RecordReplay/devtools/blob/4673a3e7a0dd24ccbcd7b37f539c5a72c8779641/src/devtools/client/debugger/src/utils/editor/source-editor.js#L4

**Reply by ryota-murakami** [reply]
@jasonLaster Thank you for super fast response! And I have very interest this. 😄
I'll dig source to see way of shipping togglable experimental feature(like Redux-DevTools), and I will added text size option (normal, small, large) simple way.

I'll open PR when the font size change working on my local machine's screen!

**Reply by markerikson** [reply]
@ryota-murakami : yeah, the changes needed to add a new experimental feature and pref are unfortunately complex :(

See this commit for all the files I had to touch:

[`c60fbdc` (#6601)](https://github.com/RecordReplay/devtools/pull/6601/commits/c60fbdcb05cb4210e2a6346dc266c06a8ebdbad9)

and I based that on the changes Jason showed in this replay and loom:

https://app.replay.io/recording/enabling-feature-flags--cba66521-7a32-4bdb-a517-447d759289ee?point=649037107325489988556295434668741&time=275.6495299751577&hasFrames=true

https://www.loom.com/share/a885f8b4bda941509fb0633a7adb8bee

All that said: I'm not sure if we need to make this an "Experimental Setting", actually.  It probably goes somewhere in the "Settings" panel overall, but I think we're just talking about a "normal" setting, not "experimental".

**Comment by ryota-murakami** [comment]
@jasonLaster @markerikson I submitted PR, thank you for guide me about what files should I edit, where is the Editor source code!
- https://github.com/RecordReplay/devtools/pull/6867
